target/linux/ar71xx/base-files/etc/hotplug.d/net/10-ar922x-led-fix

```diff
@@ -36,7 +39,7 @@ ar922x_disable_gpio_jtag() {
 	echo $regval > /sys/kernel/debug/ieee80211/$1/ath9k/regval
 }
 
-[ $phyname -a $ACTION = "add" ] && {
+[ -n $phyname -a $ACTION = "add" ] && {
```

Comment:
This is working for you?

Reply:
Maybe I shouldn't touch any other code, dismiss it! thanks!